src/SAML2/Response/AssertionAdapter.php

```diff
@@ -65,20 +60,6 @@ public function getNameID()
     }
 
     /**
-     * Attempt to get an attribute from the assertion.
-     *
-     * @deprecated use getAttributeValue instead, to be removed in 2.0
-     *
-     * @param string $name
-     * @param null   $default
-     * @return null|string[]
-     */
-    public function getAttribute($name, $default = null)
-    {
-        return $this->getAttributeValue($name, $default);
-    }
-
-    /**
      * @param string $attributeName
      * @param null   $defaultValue
      * @return null|string[]
```

Comment:
While you're at it: `$defaultValue`'s type should be `mixed|null`. The return value must consequently be changed to `@return string[]|mixed`. Or maybe we should require `$defaultValue` to be an array of strings.